views/index.handlebars

```diff
@@ -84,10 +84,15 @@
 				<div class="item active">
 					<img class="img-responsive" src="images/slider/slider0.png" alt="slider">						
 					<div class="carousel-caption">
-						<h2>Download our app for free</h2>
-						<h4>Available on App Store and Google Play</h4>
+						<!-- <h2>bic2cal challenges!</h2> -->
+						<h4>Compete <b>or</b> collaborate!</h4>
```

Comment:
I have only changed the carousel caption on one slide. Will change on others based on feedback.